src/index.ts

```diff
@@ -343,6 +349,13 @@ export class ProjectConfig {
         this.builds = this.builds.map(applyBuildPreset);
       }
     }
+
+    /**
+     * autoBasePath
+     */
+    if (options.autoBasePath) {
+      this.autoBasePath = options.autoBasePath;
```

Comment:
Hmm, maybe we should just update all the builds here. I don't see why not. Then we won't need https://github.com/Polymer/polymer-build/pull/279 at all.

Reply:
Updated!